### DISCUSSION TITLE
Order of violin plot

### DISCUSSION THREAD

**Discussion by amufaamo** [satijalab/seurat#5753] [Q&A] [answered]
I made violin plot with VlnPlot function.

But I want to change order of plot. 
For example, the order of violin what I made is ("Healthy1", "Disease1", "Healthy2", "Disease2"), but I want to re-order like ("Healthy1", "Healthy2","Disease1" , "Disease2").

I found same issue below
https://github.com/satijalab/seurat/issues/454

But my object does not have object@ident. My object has object@active.ident, but it did not work.

**Comment by samuel-marsh** [comment, marked as answer]
Hi,

The solution in that issue for V3+ of Seurat works for me.  Did you try this:

```
my_levels <- c("Healthy1", "Healthy2","Disease1" , "Disease2")
Idents(OBJ) <- factor(Idents(OBJ), levels= my_levels)
```


Best,
Sam

**Reply by amufaamo** [reply]
Thank you so much!! It worked!!
